Makefile

```diff
@@ -281,6 +281,10 @@ aer-release: $(GEN_RS) $(GEN_TS) aer-lint aer-test ## Lint, test, and install ae
 
 # Commands for other crates
 
+.PHONY: ebpf
+ebpf:
+	$(cargo) xtask build-ebpf
```

Comment:
i'm not sure the alias is adding much here.. explicit is nice :)

is it useful for local dev to have it? i assume not if we can `make ebpf`.

Reply:
I started with `xtask` but can probably get rid of it now it's in the `Makefile`. Will have to clean this up anyways as we should probably have the `auraed` build depend on the `ebpf` target.